html/twitter.html

```diff
@@ -0,0 +1,83 @@
+<html lang="en">
```

Comment:
general:
- navbar buttons: make hover wider
- mouse pointer
- add hover to users
- make followees border shorter
- make users window wider